backend/siarnaq/api/teams/views.py

```diff
@@ -173,3 +174,35 @@ def avatar(self, request, pk=None, *, episode_id):
             titan.upload_image(avatar, profile.get_avatar_path())
 
         return Response(None, status=status.HTTP_204_NO_CONTENT)
+
+    @action(
+        detail=False,
+        methods=["post"],
+        serializer_class=None,
+        permission_classes=(IsAuthenticated, IsEpisodeAvailable),
+    )
+    def record(self, request, pk=None, *, episode_id):
```

Comment:
this is wildly inefficient. It's going to recaluculate on every query, but I didn't want to mess with the db

Reply:
I think that's probably fine, maybe with some `prefetch_related`'s. At least don't worry about it for now.

Reply:
Oh, I see now that you're manually looping over the queryset with a python `for`. Yes, we should avoid that. You could use the `.count()` member function of a queryset to get its length (https://docs.djangoproject.com/en/4.1/topics/db/aggregation/), and do this for two querysets, one filtered for wins and another filter for losses.